docs/reference/errors-and-warnings/NU5130.md

```diff
@@ -0,0 +1,47 @@
+---
+title: NuGet Warning NU5130
+description: NU5130 Warning code
+author: zivkan
+ms.date: 09/16/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords:
+  - NU5130
+---
+
+# NuGet Warning NU5130
+
+<pre>Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder have compatible matches, but not exact matches in the other location. Unless intentional, consult the list of actions below:
+- Add a dependency group for .NETFramework4.7.2 to the nuspec</pre>
+
+### Issue
+
+The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the [Target Framework Moniker (TFM)](../target-frameworks.md) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers to believe the package is not compatible with the TFM. If the assemblies for the specified TFM have different dependencies to the compatible TFM defined in the dependencies group, the project using the package may experience runtime failures.
+
+### Solution
+
+* Run pack on the project
+
+If possible, use [NuGet's MSBuild pack task](../msbuild-targets.md), as it automatically matches assembly TFMs with dependency groups from the project's target frameworks. Note that your project must use `PackageReference` for its own NuGet dependencies. If your project uses packages.config, you need to use `nuget.exe pack` and a `nuspec` file.
+
+* Manually edited `nuspec` file
+
+If you are using a custom `nuspec` file, we recommend each TFM for which `lib/` or `ref/` assemblies exist should have a matching dependency group, even if the dependencies are the same as the next compatible TFM. For example, if a package contains `netstandard1.0` and `netstandard2.0` assemblies, and the dependencies are the same for both, we recommend both TFMs be listed as dependency groups with duplicate dependency items.
+
+Note that the TFM identifier used in the assembly paths use a different format to the TFM identifier used in dependency groups. The warning message specifies the correct name to use in the dependency group. If your package does not have any dependencies for that target framework, use an empty group. For example:
+
+```xml
+<package>
+  <metadata>
+    ...
+    <dependencies>
+      <group targetFramework=".NETFramework4.7.2" />
+    </dependencies>
+  </metadata>
+  ...
+<package>
+```
+
+* Remove the `lib/` or `ref/` files
```

Comment:
Do `<h4/>` titles for the solutions would be better?

Reply:
I prefer the unordered list. I think it clearly shows they're different solution options, whereas using a level 4 header I think is less clear.

Compare this: 
![image](https://user-images.githubusercontent.com/5030577/65445118-60eef380-dde6-11e9-971b-5b94624ceb8b.png)

to this:
![image](https://user-images.githubusercontent.com/5030577/65445149-6ba98880-dde6-11e9-884d-7305c09c6e29.png)